main.go

```diff
@@ -67,6 +67,7 @@ func setupFlags(app *kingpin.Application) {
 	app.Flag("errors", "Only show errors.").BoolVar(&config.Errors)
 	app.Flag("json", "Generate structured JSON rather than standard line-based output.").BoolVar(&config.JSON)
 	app.Flag("checkstyle", "Generate checkstyle XML rather than standard line-based output.").BoolVar(&config.Checkstyle)
+	app.Flag("checkstyle-console", "Generate checkstyle XML and also standard line-based output.").BoolVar(&config.CheckstyleAndConsole)
```

Comment:
@alecthomas  for v3, what do you think about changing all the output flags to be `--output=format`, so this could be accomplished with `--output=checkstyle --output=human`.

`--json` becomes `--output=json`. We could also potentially support `--output=format=filename`, so that a user could send one or more formats to a file, and one format to stdout.

Reply:
I agree with @dnephin. In my office we need checkstyle output for sonar analyze and also need output for human.